### DISCUSSION TITLE
Can not Authenticate the App

### DISCUSSION THREAD

**Discussion by MasilelaXS** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#689] [Q&A] [answered]
I hope you are all well
I am able to select the "Key File", save and relaunch. The screen below then appears and a new browser tab opens.
![Screenshot (21)](https://user-images.githubusercontent.com/38097947/160245887-359ffedf-c08c-4523-a6fa-fdbf537b97c8.png)

On the process of authentication. After I click "continue" On the screen below.
![Screenshot (23)](https://user-images.githubusercontent.com/38097947/160246084-89cccbcc-5f2b-4469-a0d9-efb31b201c97.png)

Then thereafter I am shown this error. (with the url pointing to my localhost)
![Screenshot (24)](https://user-images.githubusercontent.com/38097947/160246112-f985c3df-c823-4c74-aeff-205e46f02363.png)

I have tried this on several different machines and email address but I still get the same result. How can I fix this?

Your answers are appreciated in advance

**Comment by im-coder-lg** [comment, marked as answer]
https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/684#issuecomment-1078551891

Solved by this.

**Reply by MasilelaXS** [reply]
Thank you very much.
